content/en/blog/2022/exponential-histograms/index.md

```diff
@@ -0,0 +1,308 @@
+the questions you can answer when data is structured like this. Assuming you're
+using a histogram to track the number of milliseconds it took to respond to a
+request, you can determine:
+
+- The number of requests.
+- The minimum, maximum, and average request latency.
+- The percentage of requests that had latency less than a particular bucket
+  boundary. For example, if buckets boundaries are `[0, 5, 10]`, you can take
+  the sum of the counts of buckets `(-∞, 0], (0,5], (5,10]`, and divide by the
+  total count to determine the percentage of requests that took less than 10
+  milliseconds. If you have an SLA that 99% of requests must be resolved in more
+  than 10 milliseconds, you can determine whether or not you met it.
+- Patterns, by analyzing the distribution. For example, you might find that most
+  requests resolve quickly but a small number of requests take a long time and
+  bring down the average.
+
+The second type of OpenTelemetry histogram is the [exponential bucket
+histogram][]. Exponential bucket histograms have buckets and bucket counts, but
+instead of explicitly defining the bucket boundaries, the boundaries are
+computed based on an exponential scale. More specifically, each bucket is
+defined by an index `i` and has bucket boundaries `(base**i, base**(i+1)]`. The
+base is derived from a scale factor that is adjustable to reflect the range of
+reported measurements and is equal to `2**2**-scale`. Bucket indexes must be
+continuous, but a non-zero positive or negative offset can be defined. For
+example, at scale 0, `base = 2**2**-0 = 2` , and the bucket boundaries for
+indexes `[-2, 2]` are defined as `(.25, .5],(.5, 1],(1,2],(2,4],(4,8]`. By
+adjusting the scale, you can represent both large and small values. Like
+explicit bucket histograms, exponential bucket histograms also track the sum of
+all values, the count of all values, the maximum value, and the minimum value.
+See the [opentelemetry-proto][exponential bucket histogram proto] for the
+complete definition.
+
+## Why use exponential bucket histograms
+
+On the surface, exponential bucket histograms don't seem very different from
+explicit bucket histograms. In reality, their subtle differences yield
+dramatically different results.
+
+**Exponential bucket histograms are a more compressed representation.** Explicit
+bucket histograms encode data with a list of bucket counts and a list of `N - 1`
+bucket boundaries, where `N` is the number of buckets. Each bucket count and
+bucket boundary is an 8-byte value, so an `N` bucket explicit bucket histogram
+is encoded as `2N - 1` 8-byte values.
+
+In contrast, bucket boundaries for exponential bucket histograms are computed
+based on a scale factor and an offset defining the starting index of the
+buckets. Each bucket count is an 8-byte value, so an `N` bucket exponential
+bucket histogram is encoded as `N + 2` 8-byte values (`N` bucket counts and `2`
```

Comment:
Here and elsewhere, my _preference_ would be for the "math" to be at least in plain text, rather than code font. If you do want to style it, then (math) variables should be in italics. See the suggestion below. 

But I understand that it might not be worth it at this point, so I leave it up to you @jack-berg.

In particular, why is the (plain) number two in code font?

```suggestion
buckets. Each bucket count is an 8-byte value, so an _N_ bucket exponential
bucket histogram is encoded as (_N_+2) 8-byte values (_N_ bucket counts and 2
```

Which looks like this:

> <img width="603" alt="image" src="https://user-images.githubusercontent.com/4140793/186254372-956299fc-a989-4f12-9c24-e757ca0a250f.png">

Reply:
@chalin I am wondering, if hugo allows some setting of math code like github markdown allows it these days. I think that especially the syntax `base**i` might not be understood by everyone, whereas $base^i$ is a notation that people should know about. 

If not, it would be good to introduce them verbally:

> and has bucket boundaries `(base**i, base**(i+1)]`, where `base**i` means that base is raised to the power of i.

Reply:
We can enable support for math via Docsy. @jack-berg - let us know how you'd like to deal with this.

Reply:
I think italics is fine, updating. I'd rather just explain that `**` means "raised to the power of" then fuss around trying to get math support.